### PR TITLE
Add Electrum/Esplora and Mempool UI to Docker Compose

### PR DESCRIPTION
**Pull request by nostrdev-com**
This PR adds Electrum (Blockstream implementation) and Mempool UI to the docker compose set-up.

### CODE REVIEW

**Review by DavidGershony — APPROVED** [review]

File: docker-angor-api/docker-compose.yml [hunk 1]
```diff
@@ -53,16 +67,23 @@ services:
       DATABASE_USERNAME: "mempool"
       DATABASE_PASSWORD: "mempool"
       STATISTICS_ENABLED: "true"
-    user: "1000:1000"
+      VIRTUAL_HOST: indexer.thedude.pro
```
Comment (dangershony) [inline docker-angor-api/docker-compose.yml:70]: The only comment I have is that we should use the `indexer.angor.io` domain on the main repo

Comment (dangershony) [reply docker-angor-api/docker-compose.yml:70]:
```suggestion
      VIRTUAL_HOST: indexer.angor.io
```

File: docker-angor-api/docker-compose.yml [hunk 2]
```diff
@@ -53,16 +67,23 @@ services:
...
+      VIRTUAL_PORT: 8999
+      VIRTUAL_PROTO: http
+      VIRTUAL_NETWORK: proxy
+      LETSENCRYPT_HOST: indexer.thedude.pro
```
Comment (dangershony) [inline docker-angor-api/docker-compose.yml:74]:
```suggestion
      LETSENCRYPT_HOST: indexer.angor.io
```

File: docker-angor-api/docker-compose.yml [hunk 3]
```diff
@@ -72,15 +93,68 @@ services:
...
+      MAINNET_ENABLED: "false"
+      ROOT_NETWORK: "signet"
+      FRONTEND_HTTP_PORT: "8080"
+      BACKEND_MAINNET_HTTP_HOST: "mempool_api"
+      LIGHTNING: "false"
+      VIRTUAL_HOST: explorer.thedude.pro
+      VIRTUAL_PORT: 8080
+      VIRTUAL_PROTO: http
+      VIRTUAL_NETWORK: proxy
+      LETSENCRYPT_HOST: explorer.thedude.pro
```
Comment (dangershony) [inline docker-angor-api/docker-compose.yml:115]:
```suggestion
      LETSENCRYPT_HOST: explorer.angor.io
```

**Review by dangershony — APPROVED** [review]

**Pull request merged**